pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -2186,31 +2184,27 @@ protected void processResults(ResultHandler handler, int flags) throws IOExcepti
 
           LOGGER.log(Level.FINEST, " <=BE EmptyQuery");
 
-        {
```

Comment:
These lines make sure the variables are local, so I would like to refrain from removing these curly braces.

Reply:
Thanks. It looks good now.